README.md

```diff
@@ -287,6 +306,13 @@ class Header extends React.Component {
 | requires              | `*`                               | `true`   | Requirement that will be used in access attempt call                                                  |
 | authorizationStrategy | `(permissions, requirement) => *` | `false`  | Function that if passed will override `authorizationStrategy` passed to nearest `PermissionsProvider` |
 
+### Hooks
+
+| Hook's name    | Arguments                                   | Description                                     |
+| -------------- | ------------------------------------------- | ----------------------------------------------- |
+| usePermissions | `null`                                      | Returns context of nearest Permissions Provider |
+| useAuthorize   | `(requirement, authorizationStrategy) => *` | Returns a value of authorization                |
```

Comment:
we could mention the fact that authorizationStrategy is not required and default will be used if not passed.